doc/articles/features/status-bar-theme-color.md

```diff
@@ -0,0 +1,148 @@
+    ```cs
+    public sealed partial class App : Application
+    {
+        private UISettings _uiSettings;
+        private bool? _wasDarkMode = null;
+
+        /* ... */
+
+        protected override void OnLaunched(LaunchActivatedEventArgs e)
+        {
+            /* existing code... */
+
+            // add this line at the end of OnLaunched:
+            ConfigureStatusBar();
+        }
+
+        /* ... */
+
+        private void ConfigureStatusBar()
+        {
+            // Listen for the system theme changes.
+            _uiSettings = new UISettings();
+            _uiSettings.ColorValuesChanged += (s, e) => UpdateStatusBar();
+
+    #if __IOS__
+            // Force an update when the app is launched.
+            UpdateStatusBar();
+    #endif
+        }
+    }
+    ```
+
+7. Implement the `UpdateStatusBar()` method:
+    ```cs
+    private void UpdateStatusBar()
+    {
+        // === 1. Determine the current theme from the background value,
+        // which is calculated from the theme and can only be black or white.
+        var backgroundColor = _uiSettings.GetColorValue(UIColorType.Background);
+        var isDarkMode = backgroundColor == Windows.UI.Colors.Black;
+
+    #if __ANDROID__
+        // Prevent deadlock as setting StatusBar.ForegroundColor will also trigger this event.
+        if (_wasDarkMode == isDarkMode) return;
+        _wasDarkMode = isDarkMode;
+    #endif
+
+    #if __IOS__ || __ANDROID__
+        // === 2. Set the foreground color.
+        // note: The foreground color can only be set to a "dark/light" value. See uno remarks on StatusBar.ForegroundColor.
+        // note: For ios in dark mode, setting this value will have no effect.
+        var foreground = isDarkMode ? Windows.UI.Colors.White : Windows.UI.Colors.Black;
+        Windows.UI.ViewManagement.StatusBar.GetForCurrentView().ForegroundColor = foreground;
+
+        // === 3. Set the background color.
+        var background = isDarkMode ? Windows.UI.Colors.MidnightBlue : Windows.UI.Colors.SkyBlue;
+    #if __ANDROID__
+        // On Android, this is done by calling Window.SetStatusBarColor.
+        if (Uno.UI.ContextHelper.Current is Android.App.Activity activity)
+        {
+            activity.Window.SetStatusBarColor(background);
+        }
+    #endif
+        // On iOS, this is done via the native CommandBar which goes under the status bar.
+        // For android, we will also update the CommandBar just for consistency.
+        if (MainPage.Instance.GetCommandBar() is CommandBar commandBar)
+        {
+            commandBar.Foreground = new SolidColorBrush(foreground); // controls the color for the "MainPage" page title
+            commandBar.Background = new SolidColorBrush(background);
+        }
+    #endif
+    }
+
+## Concluding remarks
+
+This is just one way to show how it could be done. Alternatively, depending on the application, the follow options can also be considered:
```

Comment:
```suggestion
This is just one way to show how it could be done. Alternatively, depending on the application, the following options can also be considered:
```